app/.env.example

```diff
@@ -56,4 +56,4 @@ API_KEY=XXXXXXXXXXXXXXXXXXXXXX
 #SENTRY_PROJECT=XXXXXXXXXXXXXX
 
 # api key for retrieving and topping up sms4sats account balance
-#SMS_FOR_SATS_API_KEY=XXXXXXXXXXXXXXX
\ No newline at end of file
+#SMS_FOR_SATS_API_KEY=XXXXXXXXXXXXXXX
```

Comment:
Not sure where this comes from. I didn't change it.

Reply:
Maybe your IDE has different save settings regarding newlines at the end of a file.